terraform/roles.tf

```diff
@@ -251,6 +251,16 @@ data aws_iam_policy_document caascad_operator_policy {
       values   = ["true"]
     }
   }
+
+// Authorize creating requests in AWS support
+  statement {
+    actions = [
+      "support:*",
+    ]
+    resources = [
+      "*"
+    ]
+  }
```

Comment:
Add this in the already existing statement line 177

Reply:
done